src/components/SwipeClassifier.js

```diff
@@ -73,6 +89,8 @@ export class SwipeClassifier extends React.Component {
     const renderClassifierOrTutorial = () => {
       const key = this.props.workflow.first_task //always just one task
       const task = this.props.workflow.tasks[key]
+      const answers = reverse(task.answers)  //Yes is listed first in project, but we need No listed first (on left)
```

Comment:
Does this reflect what we are asking researchers to do on the project builder? (making "yes" the first answer and "no" the second answer)